docs/articles/rag_hr_chatbot.md

```diff
@@ -103,50 +111,49 @@ TOP_N = 10
 Now, you load the data.
 
 ```python
-# the new HR policy document
-r_new = requests.get(
-   "https://storage.googleapis.com/superlinked-notebook-hr-knowledgebase/hr_rag_knowledgebase.txt"
+text_df = pd.read_csv(
+    "https://storage.googleapis.com/superlinked-notebook-hr-knowledgebase/hr_rag_database.csv"
 )
-r_new.encoding = "utf-8-sig"
-text_new = r_new.text.replace("\r\n", "\n").split("\n")
-r_old = requests.get(
-   "https://storage.googleapis.com/superlinked-notebook-hr-knowledgebase/hr_rag_old_text.txt"
-)
-r_old.encoding = "utf-8-sig"
-text_old = r_old.text.replace("\r\n", "\n").split("\n")
-
-
-text_df = pd.DataFrame(text_new + text_old, columns=["body"]).reset_index()
-# add timestamps to differentiate the two sources
-text_df["creation_date"] = [int(datetime(2024, 1, 1).timestamp())] * len(text_new) + [
-   int(datetime(2023, 1, 1).timestamp())
-] * len(text_old)
 text_df.head()
 ```
 
+Let's take a sneak peak at the first five rows of our data, which comes from all three policy documents, and has the following columns:
+
+- index
+- body: text of the HR policy paragraph
+- creation_date: utc timestamp when the policy was issued
+- usefulness: ratio of people who (when prompted) deem the paragraph useful
```

Comment:
people = employees? this is a human review we have access to as part of the source data?

Reply:
yes